css/tocbot-style.css

```diff
@@ -1,8 +1,5 @@
-.js-toc-block {
-  position: relative;
-}
```

Comment:
I'm not sure if this definition should be removed. It was introduced in https://github.com/backdrop-contrib/tocbot/issues/44, and I guess we still need it to not revert the fix for that issue.

Reply:
Oops - trying to consolidate too much and forgot why that was there.